src/main/java/io/cdap/plugin/servicenow/source/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -89,14 +94,23 @@ public String getAccessToken() throws OAuthSystemException, OAuthProblemExceptio
     try {
       String accessToken = getAccessToken();
       requestBuilder.setAuthHeader(accessToken);
+      LOG.debug("Request URL: {} ", requestBuilder.build().getUrl());
+      long start = System.currentTimeMillis();
       apiResponse = executeGet(requestBuilder.build());
+      long end = System.currentTimeMillis();
+      LOG.debug("restAPI execution time for {} to {} records took {}s ", offset,
```

Comment:
we can remove logging added for debugging purposes

Reply:
Removed the logger ecfef61ba226b88cb299cac832d75aa7624808ae